modules/project/variables.tf

```diff
@@ -46,6 +46,12 @@ variable "descriptive_name" {
   default     = null
 }
 
+variable "default_service_account_setting" {
```

Comment:
nit: would `default_service_accounts` work the same while being shorter?

Reply:
yes . you mean, just rename the variable name to be shorter, right?

Reply:
yep! not critical, but the shorter the better in general.

Reply:
Done